balancer/grpclb/grpclb_remote_balancer.go

```diff
@@ -63,7 +63,7 @@ func (lb *lbBalancer) processServerList(l *lbpb.ServerList) {
 	}
 	lb.fullServerList = l.Servers
 
-	var backendAddrs []resolver.Address
+	backendAddrs := make([]resolver.Address, 0, len(l.Servers))
```

Comment:
There are cases where we end up *not* adding an entry to the slice from within the `for` loop and instead `continue`. So, it would be better to keep this as is.